src/pynwb/__init__.py

```diff
@@ -18,6 +18,35 @@
 from .spec import NWBDatasetSpec, NWBGroupSpec, NWBNamespace  # noqa E402
 from .validate import validate  # noqa: F401, E402
 
+from .termset_config import NWBTermSetConfigurator
+
+
+nwb_config = NWBTermSetConfigurator(path='src/pynwb/config/nwb_config.yaml')
```

Comment:
I know this is still in development. I think ultimately this path should be relative to the pynwb module (i.e. ``path='config/nwb_config.yml'`) not the structure in the git repo for this to work with pip package, similar to how the schema is being installed as an asset: 

https://github.com/NeurodataWithoutBorders/pynwb/blob/dfc19bd8a7576d454013ca74132babd4a42d32f4/src/pynwb/__init__.py#L31

Reply:
I appreciate this

Reply:
I just pushed an alternative approach. ``path='config/nwb_config.yml'` leads to issues importing within docs.

Reply:
I believe that should work. I think you'll  also need to update `setup.py` to pick this up when creating the python package for pip/conda. 

https://github.com/NeurodataWithoutBorders/pynwb/blob/dfc19bd8a7576d454013ca74132babd4a42d32f4/setup.py#L46